content/pt/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,738 @@
+
+No passado, a notação `volume.beta.kubernetes.io/storage-class` era utilizada no lugar do atributo `storageClassName`. Essa notação ainda funciona. Contudo, ela será totalmente depreciada numa futura versão do Kubernetes.
+
+### Política de Retenção
+
+Atualmente as políticas de retenção são:
+
+* Retain -- recuperação manual
+* Recycle -- limpeza básica (`rm -rf /thevolume/*`)
+* Delete -- o volume de armazenamento associado, como AWS EBS, GCE PD, Azure Disk ou OpenStack Cinder é deletado
+
+Atualmente, somente NFS e HostPath suportam reciclagem. Volumes AWS EBS, GCE PD, Azure Disk e Cinder suportam delete.
+
+### Opções de Montagem
+
+Um administrador do Kubernetes pode especificar opções de montagem adicionais quando um Volume Persistente é montado num nó.
+
+{{< note >}}
+Nem todos os tipos de Volume Persistente suportam opções de montagem.
+{{< /note >}}
+
+Seguem os tipos de volumes que suportam opções de montagem.
+
+* AWSElasticBlockStore
+* AzureDisk
+* AzureFile
+* CephFS
+* Cinder (OpenStack block storage)
+* GCEPersistentDisk
+* Glusterfs
+* NFS
+* Quobyte Volumes
+* RBD (Ceph Block Device)
+* StorageOS
+* VsphereVolume
+* iSCSI
+
+Não há validação em relação às opções de montagem. A montagem irá falhar se houver alguma opção inválida.
+
+No passado, a notação `volume.beta.kubernetes.io/mount-options` era usada no lugar do atributo `mountOptions`. Essa notação ainda funciona. Contudo, ela será totalmente depreciada numa futura versão do Kubernetes.
+
+### Afinidade de Nó
+
+{{< note >}}
+Para a maioria dos tipos de volume, a configuração desse campo não se faz necessária. Isso é automaticamente populado pelos seguintes volumes de bloco do tipo: [AWS EBS](/docs/concepts/storage/volumes/#awselasticblockstore), [GCE PD](/docs/concepts/storage/volumes/#gcepersistentdisk) e [Azure Disk](/docs/concepts/storage/volumes/#azuredisk). Você precisa deixar isso configurado para volumes do tipo [local](/docs/concepts/storage/volumes/#local).
+{{< /note >}}
+
+Um PV pode especificar uma [afinidade de nó](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volumenodeaffinity-v1-core) para definir restrições em relação ao limite de nós que podem acessar esse volume. Pods que utilizam um PV serão somente reservados para nós selecionados pela afinidade de nó.
+
+### Estado
+
+Um volume sempre estará em dos seguintes estados:
+
+* Available -- um recurso que está livre e ainda não foi atrelado a nenhuma requisição
+* Bound -- um volume atrelado a uma requisição
+* Released -- a requisição foi deletada, mas o curso ainda não foi recuperado pelo cluster 
```

Comment:
"deletada" to "excluída"
Please, can you make double-check the original text to the word "curso"? Not make sense. Thanks